dev/reference/configuration/tidb-server/tidb-specific-variables.md

```diff
@@ -338,6 +338,12 @@ set @@global.tidb_distsql_scan_concurrency = 10
 - Default value: 512
 - This variable is used to set the number of retries when the DDL operation fails. When the number of retries exceeds the parameter value, the wrong DDL operation is canceled.
 
+### tidb_max_delta_schema_count
```

Comment:
```suggestion
### tidb_max_delta_schema_count <span class="version-mark">New in v2.1.18 and v3.0.5</span>
```

Reply:
@zimulala I guess `tidb_max_delta_schema_count` was added [in v3.0.5](https://pingcap.com/docs-cn/v3.0/releases/3.0.5/#tidb) and v2.1.18. Right?

Reply:
@lilin90 Yes.